src/ontology/mondo-edit.obo

```diff
@@ -462876,6 +462878,3034 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0004638 ! cross-species analog lymphosarcoma
+
+[Term]
+id: MONDO:1010210
+name: malignant histiocytosis, non-human animal
+def: "This is a familial form of cancer." [OMIA:000620]
+synonym: "Histiocytic sarcoma (HS)" EXACT [OMIA:000620]
+xref: OMIA:000620
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0004612 ! cross-species analog malignant histiocytosis
+
+[Term]
+id: MONDO:1010211
+name: mycosis fungoides, non-human animal
+xref: OMIA:001109
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009691 ! cross-species analog mycosis fungoides
+
+[Term]
+id: MONDO:1010212
+name: osteosarcoma, non-human animal
+xref: OMIA:001441
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009807 ! cross-species analog osteosarcoma
+
+[Term]
+id: MONDO:1010213
+name: chronic monocytic leukemia, non-human animal
+xref: OMIA:001573
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0004614 ! cross-species analog chronic monocytic leukemia
+
+[Term]
+id: MONDO:1010214
+name: non-hodgkin lymphoma, non-human animal
+xref: OMIA:001708
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0018908 ! cross-species analog non-Hodgkin lymphoma
+
+[Term]
+id: MONDO:1010215
+name: familial adenomatous polyposis, non-human animal
+xref: OMIA:001916
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0021056 ! cross-species analog familial adenomatous polyposis 1
+
+[Term]
+id: MONDO:1010216
+name: embryonal rhabdomyosarcoma, non-human animal
+xref: OMIA:001921
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009993 ! cross-species analog embryonal rhabdomyosarcoma
+
+[Term]
+id: MONDO:1010217
+name: schwannomatosis, non-human animal
+synonym: "Peripheral nerve sheath tumors (PNST)" EXACT [OMIA:001963]
+xref: OMIA:001963
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008075 ! cross-species analog schwannomatosis
+
+[Term]
+id: MONDO:1010218
+name: glioma susceptibility, non-human animal
+xref: OMIA:002012
+is_a: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0005583 ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0100242 ! cross-species analog glioma susceptibility
```

Comment:
I removed most of the "susceptibility" terms that appear in OMIA from my list to include in Mondo per our conversations. This term slipped through. I am fine to omit this term from OMIA because our agreed upon plan/documentation it to exclude non-human animal disease susceptibility terms for now. What do you think?

Reply:
I forgot that we had discussed omitting the disease susceptibility for now. Your plan sounds good!